kpatch-build/create-diff-object.c

```diff
@@ -922,6 +1020,9 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 				continue;
 			}
 
+#ifdef __powerpc__
+			add_off = 0;
+#else
```

Comment:
Can you please point me to test case, which uncovered the issue on x86 ?. I have included the sanity check in the next version.